pro/src/pages/AdageIframe/app/components/AdageHeader/AdageHeader.tsx

```diff
@@ -57,21 +67,19 @@ export const AdageHeader = () => {
           <SvgIcon
             src={logoPassCultureIcon}
             alt="Logo du pass Culture"
-            width="120"
+            width="109"
             viewBox="0 0 71 24"
           />
         </div>
-        <AdageHeaderMenu adageUser={adageUser} />
+        <AdageHeaderMenu
+          adageUser={adageUser}
+          logAdageLinkClick={logAdageLinkClick}
+        />
         {!isLoading && (
-          <div className={styles['adage-header-nav-menu-budget']}>
-            <div className={styles['adage-header-separator']}></div>
-            <div className={styles['adage-budget-text']}>
-              Solde prévisionnel
-              <span className={styles['adage-header-budget']}>
-                {formatPrice(institutionBudget)}
-              </span>
-            </div>
-          </div>
+          <AdageHeaderBudget
+            institutionBudget={institutionBudget}
+            logAdageLinkClick={logAdageLinkClick}
+          ></AdageHeaderBudget>
```

Comment:
Tjrs self close les elements qui n'ont pas de children :grin: 
On pourrait activer ça tiens : https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/self-closing-comp.md

Reply:
Décidément fait que je me le mette dans la tête :)

Reply:
Noté pour le procaine parking de discuter l'ajout de eslint-plugin-react!